crates/sui-types/src/crypto.rs

```diff
@@ -1685,15 +1678,16 @@ pub fn construct_tbls_randomness_object_message(epoch: EpochId, obj_id: &ObjectI
     msg.extend_from_slice(obj_id.as_ref());
     msg
 }
-/// Unlike `enum Signature`, `enum CompressedSignature` does not contain public key.
-#[derive(Debug, From, Clone, Serialize, Deserialize, PartialEq, Eq, JsonSchema)]
+
+/// Unlike [enum Signature], [enum CompressedSignature] does not contain public key.
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, JsonSchema)]
 pub enum CompressedSignature {
     #[schemars(with = "Base64")]
```

Comment:
I think you can remove those `schemars`

Reply:
looks like the jsonschema derive need the schemars
```
the trait `schemars::JsonSchema` is not implemented for `serde_with::As<sui_serde::Readable<fastcrypto::encoding::Base64, serde_with::Bytes>>`
```